plugins/headlamp-plugin/plugin-management/multi-plugin-management.js

```diff
@@ -0,0 +1,125 @@
+const path = require('path');
```

Comment:
the name of the folder plugin-management seems redundant. It can be something simpler 1 word like `manager` or something like that.